src/Dialogs/Weather/SkysightDialog.cpp

```diff
@@ -0,0 +1,356 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#include "SkysightDialog.hpp"
+#include "Dialogs/Message.hpp"
+#include "Dialogs/ListPicker.hpp"
+#include "Dialogs/JobDialog.hpp"
+#include "Language/Language.hpp"
+#include "Weather/Features.hpp"
+
+#ifdef HAVE_SKYSIGHT
+#include "UIGlobals.hpp"
+#include "Look/DialogLook.hpp"
+#include "Dialogs/TextEntry.hpp"
+#include "Form/Button.hpp"
+#include "Form/ButtonPanel.hpp"
+#include "Widget/ListWidget.hpp"
+#include "Widget/ButtonPanelWidget.hpp"
+#include "Weather/Skysight/Skysight.hpp"
+#include "Operation/VerboseOperationEnvironment.hpp"
+#include "util/TrivialArray.hxx"
+#include "util/StringAPI.hxx"
+#include "Renderer/TextRowRenderer.hpp"
+#include "Renderer/TwoTextRowsRenderer.hpp"
+#include "ui/event/PeriodicTimer.hpp"
+
+#include "Protection.hpp"
+#include "DataGlobals.hpp"
+#include "Interface.hpp"
+
+#include "Language/Language.hpp"
+#include "LocalPath.hpp"
+
+#include "MapWindow/OverlayBitmap.hpp"
```

Comment:
Not necessary on this commit until time/BrokenDateTime.hpp. Need at the latter commit.